src/main/java/beam/analysis/plots/PersonTravelTimeStats.java

```diff
@@ -127,6 +128,17 @@ public void resetStats() {
         hourlyPersonTravelTimes.clear();
     }
 
+    @Override
+    public Map<String, Double> getIterationSummaryStats() {
+
+        return hourlyPersonTravelTimes.entrySet().stream().collect(Collectors.toMap(
+                e -> "ptt" + CaseFormat.UPPER_UNDERSCORE.to(CaseFormat.UPPER_CAMEL, e.getKey().toString()),
```

Comment:
No! Absolutely no archaic acronyms please. These outputs are for USERS to be able to understand and interpret. So "personTravelTime" is 10000% easier to understand and use and "ptt".

Reply:
Updated